modules/apps/item-selector/item-selector-taglib/src/main/java/com/liferay/item/selector/taglib/internal/util/ItemSelectorRepositoryEntryBrowserUtil.java

```diff
@@ -261,4 +255,48 @@ private static JSONObject _createJSONObject(String key, String value) {
 		);
 	}
 
+	private static Folder _getFolder(long folderId) throws Exception {
+		Folder folder = null;
+
+		if (folderId != DLFolderConstants.DEFAULT_PARENT_FOLDER_ID) {
+			folder = DLAppServiceUtil.getFolder(folderId);
+		}
+
+		return folder;
+	}
+
+	private static Group _getGroup(
+			Folder folder, HttpServletRequest httpServletRequest,
+			long repositoryId)
+		throws Exception {
+
+		ThemeDisplay themeDisplay =
+			(ThemeDisplay)httpServletRequest.getAttribute(
+				WebKeys.THEME_DISPLAY);
+
+		Group group = themeDisplay.getScopeGroup();
+
+		if (repositoryId != 0) {
+			try {
+				group = GroupServiceUtil.getGroup(repositoryId);
+			}
+			catch (PortalException portalException) {
+				_log.error(portalException);
```

Comment:
Shouldn't this be a warning or info level? This exception is expected when repositoryId == groupId.

Reply:
You are right. I just copied this method from another class without thinking about it. I turned it into an info message.